heartbeat/pgsql

```diff
@@ -107,6 +108,7 @@ OCF_RESKEY_replication_slot_name_default=""
 : ${OCF_RESKEY_crm_attr_timeout=${OCF_RESKEY_crm_attr_timeout_default}}
 : ${OCF_RESKEY_stop_escalate_in_slave=${OCF_RESKEY_stop_escalate_in_slave_default}}
 : ${OCF_RESKEY_replication_slot_name=${OCF_RESKEY_replication_slot_name_default}}
+: ${OCF_RESKEY_replication_interface=${OCF_RESKEY_replication_slot_interface}}
```

Comment:
Seems like you got `slot_` in here by accident, and forgot to add  `_default`.

Reply:
This is now fixed.